converters/toZigbee.js

```diff
@@ -62,6 +62,20 @@ const converters = {
             await entity.read('genOnOff', ['startUpOnOff']);
         },
     },
+    light_color_options: {
+        key: ['light_color_options'],
+        convertSet: async (entity, key, value, meta) => {
+            let optionsValue = 0;
```

Comment:
This can be simplified to `const options = value.hasOwnProperty('execute_if_off') && value.execute_if_off ? 1 : 0` (as currently nothing else is supported in the ZCL). 

Then `await entity.write('lightingColorCtrl', {options: optionsValue}, utils.getOptions(meta.mapped, entity));` can be changed to `await entity.write('lightingColorCtrl', {options}, utils.getOptions(meta.mapped, entity));`